MH4U Dex/CoreDataController.m

```diff
@@ -46,88 +48,79 @@ + (CoreDataController *)sharedCDController
 
 #pragma mark - Initial Data Loading
 
+- (void)tryLoadSequence
+{
+    if ([[NSUserDefaults standardUserDefaults] boolForKey:NeedsReloadKey]) {
+        NSLog(@"Resetting Core Data");
+        [self resetCoreData];
```

Comment:
I should have put in a TODO:, but I actually want to remove those NSLog calls once I'm sure everything works as it should be.  I tried to keep them all in that place so I could remove them a bit more easily.